### PR TITLE
Spiky juice -- functional seeds of email invite/ user invite, behind dev toggles

### PR DESCRIPTION
**Pull request by judeallred**
This PR adds 2 dev toggles, 'Email Invites' and 'Email Login':
image

Turning on Email Invites will expose the invite-user-by-email interface on the teams page:

image

( this doesn't fully work yet due to API bugs and missing UX affordances. )

Turning on 'Email Login' will expose the 'sign in with email' button on the 'Sign in' dialog, in all places that dialog can be found.:
image

This is functionally operational, but is missing all of the UI and UX work needed to polish this up for sharing with users.

### CODE REVIEW

**Comment by judeallred** [comment]
putting these live w/ dev toggles allows us to see the functionality on the live site (when we opt in to it) and start using it and improving it, as well as allows for the backend team to better work through api quirks

File: src/presenters/questions.jsx [hunk 1]
```diff
@@ -52,6 +53,7 @@ class Questions extends React.Component {
       });
     } catch (error) {
       console.error(error);
+      captureException(error);
```
Comment (judeallred) [inline src/presenters/questions.jsx]: ...this is an unrelated bugfix that was in my branch.  We don't presently capture api failures around questions unless they propagate through and crash us... this helps us get ahead of that a little more.

File: src/presenters/pop-overs/sign-in-pop.jsx [hunk 2]
```diff
@@ -26,29 +27,61 @@ const SignInPopButton = (props) => (
...
-export const SignInPop = ({header, prompt, params}) => (
+const jankyEmailPrompt = async (api) => {
+  const email = window.prompt("We'll send you a login link.\n\nWhat's your email address?");
+  if(!email) {
+    // blank or cancelled.
+    return;
+  }
+
+  try {
+    await api.post('/email/sendLoginEmail', {emailAddress:email});
```
Comment (GregWeil) [inline src/presenters/pop-overs/sign-in-pop.jsx]: There's an extra `params="hash=create-team"` prop sent to the login popover when you click the 'sign in to create a team' link. It's used by the /login/asdf routes to indicate that the create team pop should open up.

Comment (judeallred) [reply src/presenters/pop-overs/sign-in-pop.jsx]: good note, will fix! thanks.

Comment (judeallred) [reply src/presenters/pop-overs/sign-in-pop.jsx]: going to leave this be for now, but noting in a notion card:  https://www.notion.so/glitch/87bda09f85184c1d81e1659aa62e4392?v=9f7963becc9341ed9af502682c6e9eef&p=33e76165e033457c86b748f913729c5f

**Pull request merged**